### PR TITLE
Fix line break in Ubuntu installation instructions

### PR DESCRIPTION
**Pull request by robertlemke**
Signed-off-by: Robert Lemke <robert@flownative.com>

### CODE REVIEW

**Comment by nalind** [comment]
bot, test pull request

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by TomSweeneyRedHat** [comment]
bot, add author to whitelist

**Comment by TomSweeneyRedHat** [comment]
bot, retest this pleaaw

**Comment by TomSweeneyRedHat** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
@robertlemke You need to sign your PR.
git commit -a --amend -s
git push --force

Should fix this.

**Comment by rhatdan** [comment]
@robertlemke Are you still interested in fixing this?

File: install.md [hunk 1]
```diff
@@ -121,7 +121,8 @@ Then to install Buildah on Ubuntu follow the steps in this example:
   export GOPATH=`pwd`
   git clone https://github.com/projectatomic/buildah ./src/github.com/projectatomic/buildah
   cd ./src/github.com/projectatomic/buildah
-  PATH=/usr/lib/go-1.8/bin:$PATH make runc all TAGS="apparmor seccomp"
+  PATH=/usr/lib/go-1.8/bin:$PATH
```
Comment (rhatdan) [inline install.md]: This does not look correct.  The first version is correct since it will set the environment for PATH when running the make command.

**Comment by rhatdan** [comment]
I don't think this is a bug.

**Pull request closed without merging**